src/main/java/org/apache/commons/codec/binary/Base64.java

```diff
@@ -127,6 +127,58 @@ public Builder setUrlSafe(final boolean urlSafe) {
             return setEncodeTable(toUrlSafeEncodeTable(urlSafe));
         }
 
+        /**
+         * Sets the format of the decoding table.
+         * This method allows to explicitly state whether a "standard" or "URL Safe" Base64 decoding is expected.
+         * <p>
+         * Note: By default, the implementation uses the MIXED approach, allowing a seamless handling of
+         * both URL_SAFE and STANDARD base64.
```

Comment:
Use `@link` when referring to elements in this class. For example `{@link DecodeTableFormat#URL_SAFE}`.